Document/0x03-Overview.md

```diff
@@ -1,75 +1,86 @@
+## Key Areas in Mobile Application Security (AppSec)
 
-Security concerns in the mobile app space differ from traditional desktop software in some important ways. Firstly, while not many people opt to carry a desktop tower around in their pocket, doing this with a mobile device is decidedly more common. As a consequence, mobile devices are more readily lost and stolen, so adversaries are more likely to get physical access to a device and access any of the data stored.
+Many mobile application penetration testing tools have a background in network and web app penetration (app pen) testing, a quality that is valuable for mobile app testing. Almost every mobile app talks to a backend service, and those services are prone to the same kinds of attacks we are familiar with in web apps on desktop machines. Mobile apps differ in that there is a smaller attack surface and therefore more security against injection (attackers supply input to a program) and similar attacks. Instead, we must prioritize data protection on the device and the network to increase mobile security. 
 
-## Key Areas in Mobile AppSec
+Let's discuss the key areas in mobile application security.
 
-Many mobile app pen testers have a background in network and web app penetration testing, and a lot of their knowledge is useful in mobile app testing. Practically every mobile app talks to some kind of backend service, and those services are prone to the same kinds of attacks we all know and love. On the mobile app side however, there is only little attack surface for injection attacks and similar attacks. Here, the main focus shifts to data protection both on the device itself and on the network. The following are some of the key areas in mobile app security.
+>Maybe it's a good idea to number each security area below to keep it tied into this introductory sentence above? Maybe not, let me know your thoughts.
 
 ### Local Data Storage
 
-The protection of sensitive data, such as user credentials and private information, is a key focus in mobile security. Firstly, sensitive data can be unintentionally exposed to other apps running on the same device if operating system mechanisms like IPC are used improperly. Data may also unintentionally leak to cloud storage, backups, or the keyboard cache. Additionally, mobile devices can be lost or stolen more easily compared to other types of devices, so an adversary gaining physical access is a more likely scenario.
+The protection of sensitive data (such as user credentials and private information) is crucial to mobile security. If operating system mechanisms (such as inter-process communication) are used improperly, sensitive data can be exposed to other apps running on the same device. Data may also unintentionally leak to cloud storage, backups, or the keyboard cache. Additionally, mobile devices can be lost or stolen more easily compared to other types of devices, so it's more likely an individual can gain physical access to sensitive data.
```

Comment:
You use passive voice here a lot. Passive voice is not necessarily always bad, but it makes the text more boring and harder to understand.

E.g. instead of

> If operating system mechanisms (such as inter-process communication) are used improperly

something like this:

> If an app uses operating system mechanisms improperly, it can expose sensitive data...